src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ActivatorUtilities.cs

```diff
@@ -139,15 +144,17 @@ public static class ActivatorUtilities
                 // efficiently, the size savings of not using System.Linq.Expressions is more important than CPU perf.
                 return CreateFactoryReflection(instanceType, argumentTypes);
             }
+            else
```

Comment:
(nit), I'm not sure these `else` changes really add value since the `if` cases always return.

```C#
if (foo)
   return null;

// do more code
```

vs

```C#
if (foo)
   return null;
else
    // do more code
```

mean the same thing, and the latter just adds another level of indention for no real benefit (IMO).

Reply:
This was an attempt to make the code flow more clear; I don't think the linker, for example, had an issue previously but having `if (false) return; else return` seems more straightforward than `if (false) return; return`

Reply:
I double-checked the linker semantics and removed the changes. The linker under Blazor will correctly determine the case without the changes here (will remove the call to LINQ) so I removed this.